translations/harbour-seaprint-de.ts

```diff
@@ -613,1234 +613,1201 @@ auf diesem Drucker</translation>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag C5</translation>
     </message>
     <message>
         <source>17 x 22″</source>
-        <translation type="unfinished"></translation>
+        <translation>17 x 22 Zoll</translation>
     </message>
     <message>
         <source>22 x 34″</source>
-        <translation type="unfinished"></translation>
+        <translation>22 x 34 Zoll</translation>
     </message>
     <message>
         <source>34 x 44″</source>
-        <translation type="unfinished"></translation>
+        <translation>34 x 44 Zoll</translation>
     </message>
     <message>
         <source>11 x 14″</source>
-        <translation type="unfinished"></translation>
+        <translation>11 x 14 Zoll</translation>
     </message>
     <message>
         <source>12 x 14″</source>
-        <translation type="unfinished"></translation>
+        <translation>12 x 14 Zoll</translation>
     </message>
     <message>
         <source>Executive</source>
-        <translation type="unfinished"></translation>
+        <translation>Executive</translation>
     </message>
     <message>
         <source>44 x 68″</source>
-        <translation type="unfinished"></translation>
+        <translation>44 x 68 Zoll</translation>
     </message>
     <message>
         <source>European Fanfold</source>
-        <translation type="unfinished"></translation>
+        <translation>Europäische Zickzackfaltung</translation>
     </message>
     <message>
         <source>US Fanfold</source>
-        <translation type="unfinished"></translation>
+        <translation>US Zickzackfaltung</translation>
     </message>
     <message>
         <source>Foolscap</source>
-        <translation type="unfinished"></translation>
+        <translation>Foolscap</translation>
     </message>
     <message>
         <source>8 x 13″</source>
-        <translation type="unfinished"></translation>
+        <translation>8 x 13 Zoll</translation>
     </message>
     <message>
         <source>8 x 10″</source>
-        <translation type="unfinished"></translation>
+        <translation>8 x 10 Zoll</translation>
     </message>
     <message>
         <source>3 x 5″</source>
-        <translation type="unfinished"></translation>
+        <translation>3 x 5 Zoll</translation>
     </message>
     <message>
         <source>6 x 8″</source>
-        <translation type="unfinished"></translation>
+        <translation>6 x 8 Zoll</translation>
     </message>
     <message>
         <source>4 x 6″</source>
-        <translation type="unfinished"></translation>
+        <translation>4 x 6 Zoll</translation>
     </message>
     <message>
         <source>5 x 8″</source>
-        <translation type="unfinished"></translation>
+        <translation>5 x 8 Zoll</translation>
     </message>
     <message>
         <source>Statement</source>
-        <translation type="unfinished"></translation>
+        <translation>Anweisung</translation>
```

Comment:
Not convinced by this translation

Reply:
Use instead Erklärung

Reply:
Are you sure?
This refers to a paper size that is for making a statement, i..e. it's probably a bit bigger than whatever is more normal in the context.
Would anyone understand that it is the paper size being referred to?

Reply:
You are right. Statement is a US paper format. https://1a-office24.de/index.php/service/din-und-us-papierformate.html
Statement (half letter).